rcgen/tests/openssl.rs

```diff
@@ -422,11 +422,23 @@ fn test_openssl_crl_parse() {
 
 	// The properties of the CRL should match expected.
 	let openssl_issuer = X509::from_der(issuer.der()).unwrap();
-	let expected_last_update =
-		Asn1Time::from_unix(crl.params().this_update.unix_timestamp()).unwrap();
+	let expected_last_update = Asn1Time::from_unix(
+		crl.params()
+			.this_update
+			.unix_timestamp()
+			.try_into()
+			.unwrap(),
+	)
+	.unwrap();
 	assert!(openssl_crl.last_update().eq(&expected_last_update));
-	let expected_next_update =
-		Asn1Time::from_unix(crl.params().next_update.unix_timestamp()).unwrap();
+	let expected_next_update = Asn1Time::from_unix(
+		crl.params()
+			.next_update
+			.unix_timestamp()
+			.try_into()
```

Comment:
I think these `try_into()`'s are getting flagged on the 64bit build as an "unnecessary conversion": https://github.com/rustls/rcgen/actions/runs/10948893210/job/30400945346?pr=290

I would be OK with disabling the lint for both instead of trying to do a fancier gating based on arch.

Reply:
I applied a clippy allow attribute to the statements. Sorry for missing this before hitting "submit" on my own comment :)

Reply:
Np :-)